src/plugins/saved_objects/README.md

```diff
@@ -2,77 +2,78 @@
 
-## Save relationships to index pattern
+### Relationships
 
-Saved objects that have relationships to index patterns are saved using the [`kibanaSavedObjectMeta`](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/4a06f5a6fe404a65b11775d292afaff4b8677c33/src/plugins/saved_objects/public/saved_object/helpers/serialize_saved_object.ts#L59) attribute and the [`references`](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/4a06f5a6fe404a65b11775d292afaff4b8677c33/src/plugins/saved_objects/public/saved_object/helpers/serialize_saved_object.ts#L60) array structure. Functions from the data plugin are used by the saved object plugin to manage this index pattern relationship. 
+Saved objects can persist parent/child relationships to other saved objects via `references`. These relationships can be viewed on the UI in the [saved objects management plugin](src/core/server/saved_objects_management/README.md). Relationships can be useful to combine existing saved objects to produce new ones, such as using an index pattern as the source for a visualization, or a dashboard consisting of many visualizations.
 
-A standard saved object and its index pattern relationship:
+Some saved object fields have pre-defined logic. For example, if a saved object type has a `searchSource` field indicating an index pattern relationship, a reference will automatically be created using the [`kibanaSavedObjectMeta`](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/4a06f5a6fe404a65b11775d292afaff4b8677c33/src/plugins/saved_objects/public/saved_object/helpers/serialize_saved_object.ts#L59) attribute and the [`references`](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/4a06f5a6fe404a65b11775d292afaff4b8677c33/src/plugins/saved_objects/public/saved_object/helpers/serialize_saved_object.ts#L60) array structure. Functions from the data plugin are used by the saved object plugin to manage this index pattern relationship.
+
+An example of a visualization saved object and its index pattern relationship:
 
 ```ts
 
 "kibanaSavedObjectMeta" : {
     "searchSourceJSON" : """{"filter":[],"query":{"query":"","language":"kuery"},"indexRefName":"kibanaSavedObjectMeta.searchSourceJSON.index"}"""
-      }
-    },
-    "type" : "visualization",
-    "references" : [
-      {
-          "name" : "kibanaSavedObjectMeta.searchSourceJSON.index",
-          "type" : "index-pattern",
-          "id" : "90943e30-9a47-11e8-b64d-95841ca0b247"
-      }
-    ],
+}
+"type" : "visualization",
+"references" : [
+  {
+    "name" : "kibanaSavedObjectMeta.searchSourceJSON.index",
```

Comment:
Is `kibanaSavedObjectMeta` a keyword? If not, can we use our own name?

Reply:
this is just what exists currently for visualizations. it's a saved object attribute for the visualization saved obj type. i haven't changed anything here

Reply:
it is a keyword unfortunately